src/SailthruMessage.php

```diff
@@ -138,6 +138,30 @@ public function template(string $template): SailthruMessage
         return $this;
     }
 
+    /**
+     * @param array $from Associative array with email and name
+     *
+     * @return SailthruMessage
+     */
+    public function from(array $from): SailthruMessage
```

Comment:
Did you manage to test if this does anything useful?

Reply:
Hmm, lemme try. The template does specify the sender but it can also be blank.

Reply:
If it works, keep it. If not, take it out. Maybe it's also an account specific setting?